crates/bevy_ui/src/focus.rs

```diff
@@ -160,28 +161,34 @@ pub fn ui_focus_system(
     let is_ui_disabled =
         |camera_ui| matches!(camera_ui, Some(&UiCameraConfig { show_ui: false, .. }));
 
-    let cursor_position = camera
+    // If there is only one camera, we use it as default
+    let default_single_camera = camera_query.get_single().ok().map(|(entity, _, _)| entity);
+    let camera_cursor_positions: Vec<(Entity, Vec2)> = camera_query
```

Comment:
nit: this should probably be a hashmap

Reply:
To be honest, I am running out of motivation on this pull request. I was also thinking about a way to provide coordinates manually, but this requires a little more thinking and testing that I have in me currently. I would love to focus on the changes which are necessary for this to be a mergable valuable piece of work, and leave all the rest of improvements for later (and probably for someone else to work on).

Reply:
I'm willing to do a follow-up PR after this is merged.